code/modules/food_and_drinks/food/snacks_meat.dm

```diff
@@ -54,6 +54,43 @@
 	tastes = list("fish" = 1, "chips" = 1)
 	foodtype = MEAT | VEGETABLES | FRIED
 
+/obj/item/reagent_containers/food/snacks/sushie_basic
+	name = "Funa Hosomaki"
+	desc = "A small cylindrical filled with rice and fish."
+	icon = 'modular_citadel/icons/obj/food/food.dmi'
+	icon_state = "sushie_basic"
+	bonus_reagents = list("vitamin" = 2)
+	list_reagents = list("nutriment" = 4)
+	bitesize = 1
```

Comment:
Granted that Sushi is, by definition, bite-sized, I don't feel this is appropriate.